src/clue/app-config.json

```diff
@@ -78,6 +78,41 @@
           "showStars": true
         }
       ]
+    },
+    {
+      "tab": "learning-log",
+      "label": "Learning Log",
+      "sections": [
+        {
+          "title": "Logs",
+          "type": "learning-logs",
+          "dataTestHeader": "learning-log-section",
+          "dataTestItem": "learning-log-list-items",
+          "documentTypes": ["learningLog"]
+        }
+      ]
+    },
+    {
+      "tab": "supports",
+      "label": "Supports",
+      "sections": [
+        {
+          "title": "Just-in-time Supports",
+          "type": "curricular-supports",
+          "dataTestHeader": "supports-section",
+          "dataTestItem": "supports-list-items",
+          "documentTypes": ["supportPublication"],
+          "properties": ["curricularSupport"]
+        },
+        {
+          "title": "Teacher Supports",
+          "type": "teacher-supports",
+          "dataTestHeader": "supports-section",
+          "dataTestItem": "supports-list-items",
+          "documentTypes": ["supportPublication"],
+          "properties": ["teacherSupport"]
+        }
+      ]
```

Comment:
Not really related to this PR but why is this file in json format and not an exported typed constant?  We would get type checking over the config for free that way.

Reply:
I'm imagining that this may become part of authoring at some point, e.g. it could be passed in from LARA, which is easier to do with JSON.